test/unit/org/apache/cassandra/concurrent/SimulatedExecutorFactory.java

```diff
@@ -153,9 +210,92 @@ public Thread startThread(String name, Runnable runnable, InfiniteLoopExecutor.D
     }
 
     @Override
-    public Interruptible infiniteLoop(String name, Interruptible.Task task, InfiniteLoopExecutor.SimulatorSafe simulatorSafe, InfiniteLoopExecutor.Daemon daemon, InfiniteLoopExecutor.Interrupts interrupts)
+    public Interruptible infiniteLoop(String name,
```

Comment:
this change isn't needed for this patch, but figured its best to keep in-sync with the accord branch as it uses this class often